Lab1.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
I'm struggling to wade through this raw jupyter file, so bear with me. I'll just be commenting on the content such as explanations and figures instead of all the metadata and source code in this JSON file.